flink-autoscaler/src/main/java/org/apache/flink/autoscaler/JobVertexScaler.java

```diff
@@ -95,9 +105,11 @@ public int computeScaleTargetParallelism(
             LOG.warn(
                     "Target data rate is not available for {}, cannot compute new parallelism",
                     vertex);
-            return currentParallelism;
+            return VertexScalingResult.normalScaling(currentParallelism);
         }
 
+        targetCapacity *= backpropagationScaleFactor;
```

Comment:
I think we should probably not trigger a scale down just based on an identified bottleneck. We should use this logic to prevent scale ups, but triggering a scale down may be a bit too aggressive, what do you think?

Reply:
Yes, you are right, sometimes it may be too aggressive. But there are also some cases where this scale down is important. I can add an extra option for more smooth scaling down in case of bottlenecks. What do you think?

Reply:
Fixed